src/main/java/duke/commands/FindCommand.java

```diff
@@ -0,0 +1,56 @@
+package duke.commands;
+
+import duke.DukeException;
+import duke.Storage;
+import duke.Ui;
+import duke.tasks.Task;
+import duke.tasks.TaskList;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class FindCommand extends Command{
+    private static final String MISSING_ARGUMENTS = "The find command needs at least an argument.";
```

Comment:
Perhaps the error messages can be in Ui so all messages shown to users are there and it is easier for editing for other developers?